### PR TITLE
update migrations to match documentation

### PR DESCRIPTION
**Pull request by iblazevic**
updating migrations to match documentation for issue #28

### CODE REVIEW

File: migrations/20141003120222-create-user.js [hunk 1]
```diff
@@ -1,17 +1,15 @@
 "use strict";
 
 module.exports = {
-  up: function(migration, DataTypes, done) {
-    migration
+  up: function(queryInterface, Sequelize) {
+    queryInterface
```
Comment (sdepold) [inline migrations/20141003120222-create-user.js]: We need to add a `return` otherwise the promise will not be handled properly.

**Comment by sdepold** [comment]
:cool: Can you please add the `return`s in the migrations?

**Comment by iblazevic** [comment]
Added. It did work for some reason (I mean migration was applied), even on one more place, good thing its fixed now to be sure its working properly.

Thanks!

**Comment by sdepold** [comment]
:clap:

**Comment by sdepold** [comment]
It ran correctly because it actually executes the migration, but it will not wait for it to be finished

**Pull request merged**